### ISSUE TITLE
MomentJS warning on `npm start`

### ISSUE THREAD

**Issue by alanmoo** [mozilla/network#380]
If you keep an eye on the console, MomentJS gets upset when the project is starting up.

```
Deprecation warning: value provided is not in a recognized RFC2822 or ISO format. moment construction falls back to js Date(), which is not reliable across al
l browsers and versions. Non RFC2822/ISO date formats are discouraged and will be removed in an upcoming major release. Please refer to http://momentjs.com/gu
ides/#/warnings/js-date/ for more info.
```

Let's figure out why

**Comment by gvn** [comment]
I'm pretty sure this is because dates for News are coming back in this format `YYYY-MM-DD` instead of the longform ISO format.

If possible we should probably fix this at this service level.

**Comment by alanmoo** [comment]
Blocked by https://github.com/mozilla/network-api/issues/210

**Comment by alanmoo** [comment]
Or is it not blocked? I think the appropriate solution here might be to tell Moment the format of the news response- `moment(data, 'YYYY-MM-DD')` from http://momentjs.com/guides/#/parsing/known-formats/

**Comment by gvn** [comment]
I think we can force it to work, but it'd still be nice to have consistency on the API side.

**Comment by alanmoo** [comment]
I believe the reason news articles don't have a time is because the sources don't usually have a time of publish, so we didn't want to make something up.
